images/kindnetd/Dockerfile

```diff
@@ -27,6 +27,6 @@ COPY . .
 RUN go build -o /go/bin/kindnetd .
 
 # STEP 2: Build small image
-FROM scratch
+FROM gcr.io/google-containers/debian-iptables:v11.0.2
```

Comment:
we need to use the arch specific image and cross build :sweat_smile:

Reply:
I´ll need some guidance here, I couldn´t find info or some example on how to do it

Reply:
@BenTheElder done